### PR TITLE
Entity predicates javadoc

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

File: mappings/net/minecraft/predicate/entity/EntityPredicates.mapping [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+		COMMENT
+		COMMENT <p>An entity is valid when the entity is alive.
+		COMMENT
+		COMMENT @see net.minecraft.entity.Entity#isAlive()
 	FIELD field_6155 EXCEPT_SPECTATOR Ljava/util/function/Predicate;
 	FIELD field_6156 EXCEPT_CREATIVE_OR_SPECTATOR Ljava/util/function/Predicate;
 	FIELD field_6157 VALID_ENTITY_LIVING Ljava/util/function/Predicate;
+		COMMENT Tests if a living entity is valid.
+		COMMENT
+		COMMENT <p>A valid living entity must be entity is alive.
```
Comment (YanisBft) [inline mappings/net/minecraft/predicate/entity/EntityPredicates.mapping]: I would say `A living entity is valid when the living entity is alive.`

Comment (i509VCB) [reply mappings/net/minecraft/predicate/entity/EntityPredicates.mapping]: lol apparently I brainfarted my grammar there

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/predicate/entity/EntityPredicates.mapping [hunk 2]
```diff
@@ -1,11 +1,38 @@
 CLASS net/minecraft/class_1301 net/minecraft/predicate/entity/EntityPredicates
 	FIELD field_22280 EXCEPT_CREATIVE_SPECTATOR_OR_PEACEFUL Ljava/util/function/Predicate;
 	FIELD field_6152 VALID_INVENTORIES Ljava/util/function/Predicate;
+		COMMENT Tests if an entity has a valid inventory.
+		COMMENT
+		COMMENT <p>An entity has a valid inventory when:
+		COMMENT
+		COMMENT <ul><li>The entity is alive
+		COMMENT <li>The entity is an {@link java.lang.Class#isAssignableFrom(Class) instanceof} {@link net.minecraft.inventory.Inventory}
```
Comment (liach) [inline mappings/net/minecraft/predicate/entity/EntityPredicates.mapping]:
```suggestion
		COMMENT <li>The entity implements {@link net.minecraft.inventory.Inventory}
```

File: mappings/net/minecraft/predicate/entity/EntityPredicates.mapping [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
+		COMMENT </ul>
 	FIELD field_6154 VALID_ENTITY Ljava/util/function/Predicate;
+		COMMENT Tests if an entity is valid.
+		COMMENT
+		COMMENT <p>An entity is valid when the entity is alive.
+		COMMENT
+		COMMENT @see net.minecraft.entity.Entity#isAlive()
 	FIELD field_6155 EXCEPT_SPECTATOR Ljava/util/function/Predicate;
 	FIELD field_6156 EXCEPT_CREATIVE_OR_SPECTATOR Ljava/util/function/Predicate;
 	FIELD field_6157 VALID_ENTITY_LIVING Ljava/util/function/Predicate;
```
Comment (liach) [inline mappings/net/minecraft/predicate/entity/EntityPredicates.mapping]: This field deserves a rename to `VALID_LIVING_ENTITY`

**Review by Juuxel — APPROVED** [review]

**Review by valoeghese — APPROVED** [review]

**Review by Earthcomputer — APPROVED** [review]

**Pull request merged**